File: view/src/main/java/com/mackwu/view/BasicFragment.java
package com.mackwu.view;

import android.os.Bundle;
import android.view.View;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import com.mackwu.view.basic.EditTextActivity;
import com.mackwu.xmvc.BaseFragment;
import com.mackwu.xmvc.util.ActivityStartUtil;

import butterknife.OnClick;

public class BasicFragment extends BaseFragment {

    @Override
    public int getLayoutId() {
        return R.layout.fragment_basic;
    }

    @Override
    public void initView(@NonNull View view, @Nullable Bundle savedInstanceState) {

    }

    @Override
    public void initData(@Nullable Bundle savedInstanceState) {

    }

    @OnClick(R.id.btn_test)
    public void onBtnTestClicked() {
    }

    @OnClick(R.id.btn_text_view)
    public void onBtnTextViewClicked() {
    }

    @OnClick(R.id.btn_edit_text)
    public void onBtnEditTextClicked() {
        ActivityStartUtil.startActivity(this, EditTextActivity.class);
    }

    @OnClick(R.id.btn_recycler_view)
    public void onBtnRecyclerViewClicked() {
    }
}

File: view/src/main/java/com/mackwu/view/BasicFragment.kt
package com.mackwu.view

import com.mackwu.view.basic.recycler.RecyclerActivity
import com.mackwu.view.util.SimpleBaseFragment
import com.mackwu.view.util.startActivityCls
import kotlinx.android.synthetic.main.fragment_basic.*

/**
 * ===================================================
 * Created by MackWu on 2020/3/30 10:10
 * <a href="mailto:wumengjiao828@163.com">Contact me</a>
 * <a href="https://github.com/mackwu828">Follow me</a>
 * ===================================================
 */
class BasicFragment : SimpleBaseFragment() {

    override val layoutId: Int = R.layout.fragment_basic

    override fun initView() {
        btn_recycler_view.setOnClickListener { startActivityCls(RecyclerActivity::class.java) }
    }

}